src/IB/IBFEMethod.cpp

```diff
@@ -1870,6 +1870,7 @@ IBFEMethod::putToDatabase(Pointer<Database> db)
     db->putBool("d_use_jump_conditions", d_use_jump_conditions);
     db->putBool("d_use_consistent_mass_matrix", d_use_consistent_mass_matrix);
     db->putString("d_libmesh_partitioner_type", enum_to_string<LibmeshPartitionerType>(d_libmesh_partitioner_type));
+    db->putDouble("workload_quad_point_weight", d_default_workload_spec.q_point_weight);
```

Comment:
Do we get this value from the restart database? If not, we don't need to store it hre.

Reply:
This value is read in `getFromInput` so I figured we need it here too. Is this correct? I have to admit that I don't know the difference between the variable and restart databases.

Reply:
We don't really have any system for keeping track of which variables are stateful and should be stashed in the restart database, and which are stateful but should be overwritten by new input at restart. (You could imaging wanting to change some settings when you restart, but that there are other settings for which changes cannot be accommodated.)

Reply:
Any variable whose value is determined from the input database upon class creation does not need to go into the restart database. Variables that describe the fundamental state of the class, or that should not change upon restart, should go into the restart database and be set from the values stored in that database. Does that make sense?

Reply:
(Of course, if you have a variable whose value is set from the input database and then reset from the restart database --- its value needs to be stored in the restart database.)

Reply:
> Any variable whose value is determined from the input database upon class creation does not need to go into the restart database.

Sounds like I don't need to use it here, then.